doc/index.rst

```diff
@@ -2,32 +2,55 @@
 Nextstrain CLI
 ==============
 
-.. warning::
-   This is the Nextstrain CLI's reference documentation. For the general Nextstrain docs, go here: :doc:`docs.nextstrain.org:index`
-
-This is the documentation for the `Nextstrain <https://nextstrain.org>`__
-command-line interface (CLI)---a program called ``nextstrain``---which aims to
-provide a consistent way to run and visualize pathogen builds and access
-Nextstrain components like :doc:`Augur <augur:index>` and :doc:`Auspice
-<auspice:index>` across computing environments such
-as `Docker <https://docker.com>`__, `Conda
+.. hint::
+   This is reference documentation for the Nextstrain CLI (command-line
+   interface).  If you're just getting started with Nextstrain, please see
+   :doc:`our general documentation <docs:index>` instead.
+
+The Nextstrain CLI is a program called ``nextstrain``.  It aims to provide a
+consistent way to run and visualize pathogen builds and access Nextstrain
+components like :doc:`Augur <augur:index>` and :doc:`Auspice <auspice:index>`
+across computing environments such as `Docker <https://docker.com>`__, `Conda
```

Comment:
[runtime](https://docs.nextstrain.org/en/latest/reference/glossary.html#term-runtime)?

Reply:
Yes! …but I held off in this change/PR because other places in the CLI docs still using "computing environment" and doing that switch wasn't in the scope of my changes here. I definitely want the switch to "runtime" to happen, but want it to happen all at once to keep the docs internally consistent.